docs/index.html

```diff
@@ -2,30 +2,31 @@
 <html lang="en">
 
 <head>
-	<meta charset="UTF-8">
-	<title>docsify</title>
-	<link rel="icon" href="_media/favicon.ico">
-	<meta name="google-site-verification" content="6t0LoIeFksrjF4c9sqUEsVXiQNxLp2hgoqo0KryT-sE" />
-	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-	<meta name="keywords" content="doc,docs,documentation,gitbook,creator,generator,github,jekyll,github-pages">
-	<meta name="description" content="A magical documentation generator.">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
-	<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/vue.css" title="vue">
-	<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/dark.css" title="dark" disabled>
-	<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/buble.css" title="buble" disabled>
-	<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/pure.css" title="pure" disabled>
+  <meta charset="UTF-8">
+  <title>docsify</title>
+  <link rel="icon" href="_media/favicon.ico">
+  <meta name="google-site-verification" content="6t0LoIeFksrjF4c9sqUEsVXiQNxLp2hgoqo0KryT-sE" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="keywords" content="doc,docs,documentation,gitbook,creator,generator,github,jekyll,github-pages">
+  <meta name="description" content="A magical documentation generator.">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/vue.css" title="vue">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/dark.css" title="dark" disabled>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/buble.css" title="buble" disabled>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/lib/themes/pure.css" title="pure" disabled>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/vuep/dist/vuep.css">
 
-	<style>
-		nav.app-nav li ul {
-			min-width: 100px;
-		}
-	</style>
+  <style>
+    nav.app-nav li ul {
+      min-width: 100px;
+    }
+  </style>
 </head>
```

Comment:
@trusktr --
Prettier is installed as a devDependency with the project I see squiggly lines in the editor when a rule is broken, so my assumption is that it is working as configured. Since I didn't explicitly change these lines, I would expect an error to be shown both on my console and the lint checks here on GitHub. Since that isn't happening, it looks like we need to update our configuration to prevent this from happening.

Reply:
I think we should reconsider the eslint + prettier configuration, I think my workspace is creating this issue. Happened earlier as well. 

I am not sure why it's happening, we do have `.vscode` config which allows everyone to run prettier on save.

Reply:
Marking as resolved since 1) the changes here align with our prettier settings and 2) getting everyone else's prettier configuration working properly is a separate issue.